functional-tests/hoverctl/config_test.go

```diff
@@ -17,13 +17,13 @@ var _ = Describe("When I use hoverctl", func() {
 		})
 
 		It("prints the contents of the config.yaml", func() {
-			functional_tests.Run(hoverctlBinary, "targets", "create")
+			functional_tests.Run(hoverctlBinary, "targets", "new", "config-test")
```

Comment:
There were a bit discussions last week, I think at one point I mentioned using the create/update/delete subcommands.  TBH, I don't remember what the conclusion was. If this is to match the --new-target flag, that's fine.